source/js/header.js

```diff
@@ -66,6 +66,22 @@
       performOnElement('shop-badge', showElement);
     }
   }
+  
+  var setCountryIcon = function(cookies) {
+    var countryCode = 'us';
+    if(cookies.indexOf('_sp_locale=') > -1){
+      var splitCookies = cookies.split('; ');
+      for(cookieIndex = 0; cookieIndex < splitCookies.length; cookieIndex++) {
```

Comment:
Might be out of your control but is there a way to be able to just access the cookie using the key and not have to do a loop to split them up like this? Seems redundant to have key-value pairs if you're doing an index on top of that.

Reply:
I'm actually defining the cookie for the first time here, so I can change it. My thought here was the `_sp_locale` cookie would contain both the sp_country and sp_language key/value pairs.